src/main/resources/assets/robotics/lang/de_de.json

```diff
@@ -39,10 +39,10 @@
   "block.robotics.code_editor.tooltip": "PROGRAMM-SCHREIBER",
   "block.robotics.code_editor.tooltip.summary": "Ein Block, mit dem du den _Inhalt_ eines Programms _bearbeiten_ kannst!",
   "block.robotics.code_editor.tooltip.condition1": "Bei R-Klick mit einem Programm",
-  "block.robotics.code_editor.tooltip.behaviour1": "Lässt dich das _Programm verändern_",
+  "block.robotics.code_editor.tooltip.behaviour1": "Lässt sich das _Programm verändern_",

```

Comment:
It's supposed to be `Lässt dich das _Programm verändern_` as it gets put after `block.robotics.code_editor.tooltip.condition1` in-game:

> Bei R-Klick mit einem Programm:
Lässt *dich* das Programm verändern

Maybe it's not good worded, if you find a better solution let me know!